move-to-sdk/app/io/sphere/sdk/facets/RangeFacetImpl.java

```diff
@@ -0,0 +1,133 @@
+package io.sphere.sdk.facets;
+
+import io.sphere.sdk.search.FacetedSearchExpression;
+import io.sphere.sdk.search.PagedSearchResult;
+import io.sphere.sdk.search.RangeFacetResult;
+import io.sphere.sdk.search.model.FacetedSearchSearchModel;
+import io.sphere.sdk.search.model.RangeTermFacetedSearchSearchModel;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.stream.Collectors.toList;
+
+public class RangeFacetImpl<T> extends BaseFacet<T> implements RangeFacet<T> {
+
+    private final List<String> selectedValues;
```

Comment:
Not sure if `List<String>` is the best way to store these values. In the case of select facet they were a list of values (e.g. red, blue, green), but for slider range facets that would be a single range and for bucket it can be a list of ranges. In any case, I would have here a list of `FacetRange`.